### ISSUE TITLE
Mixin's arguments can't work with IE filters (and other unusual places)

### ISSUE THREAD

**Issue by devgru** [less/less.ruby#127]
I tried to use:
    .ie-opacity (@opacity: 0){
        filter: alpha(opacity=@opacity);
    }

And I always get opacity=0 ('cause it's default value), with any @opacity given.

**Comment by jtousek** [comment]
yeah this is really annoying - filter: alpha(...) is useful sometimes

**Comment by timkelty** [comment]
+1 for this!

**Comment by jtousek** [comment]
cloudhead, any progress on this? this is key feature!! I won't be able to actually use less without this one.
